stories/Title.stories.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Title } from 'components/Title';
+import { Helmet, HelmetProvider } from 'react-helmet-async';
+
+import Provider from './Provider';
+import configureStore from './configureStore';
+
+const store = configureStore({
+  pathname: '/protein/uniprot/',
+  search: '?page_size=2',
+  hash: 'table',
+});
+
+const meta = {
+  title: 'InterPro UI/Title',
+  component: Title,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+  decorators: [
+    (Story) => (
+      <Provider store={store}>
+        <Story />
+      </Provider>
+    ),
+    (Story) => (
+      <HelmetProvider>
+        <Helmet titleTemplate="%s - InterPro" defaultTitle="InterPro" />
+        <Story />
+      </HelmetProvider>
+    ),
+  ],
+} satisfies Meta<typeof Title>;
+
+export default meta;
+type TitleStory = StoryObj<typeof meta>;
+
+const databasesPayload = {
+  payload: {
+    databases: {
+      interpro: {
+        name: 'InterPro',
+      },
+      uniprot: {
+        name: 'UniProtKB',
+      },
+      reviewed: {
+        name: 'UniProtKB/Swiss-Prot',
+      },
+      pfam: {
+        name: 'Pfam',
+      },
+      pdb: {
+        name: 'PDB',
+      },
+    },
+  },
+  loading: false,
+  progress: 1,
+  ok: true,
+  status: 200,
+  url: '',
+} as unknown as RequestedData<RootAPIPayload>;
+
+export const InterproTitle: TitleStory = {
+  args: {
+    metadata: {
+      name: {
+        name: 'Kringle',
+      },
+      type: 'domain',
+      accession: 'IPR000001',
+      source_database: 'interpro',
+    } as EntryMetadata as Metadata,
+    data: databasesPayload,
+    entries: [],
+    mainType: 'entry',
+  },
+};
+export const PfamTitle: TitleStory = {
+  args: {
+    metadata: {
+      name: {
+        name: 'Piwi domain',
+      },
+      type: 'domain',
+      accession: 'PF02171',
+      source_database: 'pfam',
+    } as EntryMetadata as Metadata,
+    data: databasesPayload,
+    mainType: 'entry',
+  },
+};
+export const ProteinTitle: TitleStory = {
+  args: {
+    metadata: {
+      length: 462,
+      name: {
+        name: 'VATB_METVS',
+      },
+      accession: 'A6UP55',
+      source_database: 'reviewed',
```

Comment:
Maybe we should add a `text-transform: none;` rule to superset the one from VF:

![image](https://github.com/ProteinsWebTeam/interpro7-client/assets/2097501/4a659551-f3d6-4bce-a434-7dedde188dad)

![image](https://github.com/ProteinsWebTeam/interpro7-client/assets/2097501/cc7949f9-bdf9-4c3d-87c1-1bc1755dd270)

Reply:
ok done